kf/bpf.go

```diff
@@ -581,6 +581,9 @@ func (b *BPF) GetArtifacts(conf *config.Config) error {
 		}
 	case fileScheme:
 		{
+			if isDefaultURLUsed {
+				URL.Path = path.Join(URL.Path, b.Program.Name, b.Program.Version, platform, b.Program.Artifact)
```

Comment:
Doesn't it make sense to move this logic above the switch statement after line 551?